src/style-sheets/layout/_footer.scss

```diff
@@ -86,37 +86,35 @@ footer {
     }
 }
 
-@include medium('down') {
-    footer {
-        & .footer-nav-module {
-            width: 100%;
-            flex-direction: column;
-            align-items: center;
-
-            & .left-navigation {
-                width: 20%;
-                display: flex;
-                flex-direction: column;
-                align-items: center;
-            }
+@include xlarge('down') {
```

Comment:
Are we not taking a mobile-first approach?

Reply:
Refactored it, might need to discuss refactoring the css we have now to be mobile design first